src/Swift.Bindings/src/Demangler/TbdParser/Parsing/JsonTbdFormatParser.cs

```diff
@@ -17,21 +17,21 @@ public class JsonTbdFormatParser : TbdFormatParserBase
         /// <summary>
         /// Creates a new JSON TBD format parser
         /// </summary>
-        public JsonTbdFormatParser(ILogger logger) : base(logger)
+        public JsonTbdFormatParser(ILogger<JsonTbdFormatParser> logger) : base(logger)
```

Comment:
```suggestion
        public JsonTbdFormatParser(ILogger logger) : base(logger)
```

Per our dicussion, I think the "category" for the logger should be specified only when we're creating the logger - the thing using the logger should not care about it.

Reply:
Similarly in other places. Basically I don't see a reason to use `ILogger<T>` anywhere as an explicit type. It would make sense if we used DI, but we don't...